packages/contracts-bedrock/src/libraries/SafeCall.sol

```diff
@@ -23,18 +22,42 @@ library SafeCall {
                     0 // outlen
                 )
         }
-        return _success;
+    }
+
+    /// @notice Perform a low level call with all gas without copying any returndata
+    /// @param _target   Address to call
+    /// @param _value    Amount of value to pass to the call
+    function send(address _target, uint256 _value) internal returns (bool success_) {
+        assembly {
+            success_ :=
+                call(
+                    gas(), // gas
+                    _target, // recipient
+                    _value, // ether value
+                    0, // inloc
+                    0, // inlen
+                    0, // outloc
+                    0 // outlen
+                )
+        }
```

Comment:
nit, could also re-use the existing implementation
```suggestion
        success_ = send(_target, gasleft(), _value);
```

Reply:
Good call